src/IECoreMaya/SceneShapeSubSceneOverride.cpp

```diff
@@ -354,7 +354,12 @@ ExpandulatorPtr expandulatorGetter( const ExpandulatorCacheGetterKey &key, size_
 }
 
 using ExpandulatorCache = IECore::LRUCache<IECore::MurmurHash, ExpandulatorPtr, IECore::LRUCachePolicy::Parallel, ExpandulatorCacheGetterKey>;
-ExpandulatorCache g_expandulatorCache( expandulatorGetter, memoryLimit() * 0.25 );
+
+ExpandulatorCache &expandulatorCache()
```

Comment:
Curious, what is the advantage here?

Reply:
The clang-tidy warning (in CLion inspection, I didn't actually build with clang) was complaining that initializing `g_expandulatorCache` directly could cause a seg fault on startup.